pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -67,6 +68,8 @@ message ApiSession {
 message ApiSessionAdded {
     bool isFullState = 1;
     repeated ApiSession apiSessions = 2;
+    string syncStrategy = 3;
+    string syncState = 4;
```

Comment:
Added as an extension point for different strategies. The syncStrategy is used to denote what strategy is being used and syncState is some string that represents data that the strategy cares about (i.e. some serialized data).